chat-components/database/src/main/java/com/pubnub/components/data/message/DBMessage.kt

```diff
@@ -22,15 +19,15 @@ data class DBMessage(
     override val id: String,
     override val text: String,
     override val contentType: String? = null,
-    override val content: DBCustomContent? = null,
+    @field:TypeConverters(CustomContentConverter::class)
```

Comment:
@kazydek `custom` and `content` fields marked with annotation (to parse json to `Any` type)